src/pybind/mgr/cephadm/services/iscsi.py

```diff
@@ -37,6 +37,7 @@ def prepare_create(self, daemon_spec: CephadmDaemonSpec[IscsiServiceSpec]) -> Ce
             'caps': ['mon', 'profile rbd, '
                             'allow command "osd blocklist", '
                             'allow command "config-key get" with "key" prefix "iscsi/"',
+                     'mgr', 'allow *',
```

Comment:
Can `profile rbd` just be used here as well?

Reply:
Never mind, it looks like that profile just restricts to using the "rbd_support" module commands.

Reply:
@dillaman : I think `"service status"` is allowed with `"mgr allow"`.  In fact, if you take a look to the fixed bug, the error we have in the mgr log is related with "service status". Once added the new caps for the mgr, the error dissapear.

Reply:
Yeah, I suspect so. I was just thinking about giving the absolute minimum permissions instead of allowing the ceph-iscsi user to do anything it wants to the system.

Reply:
Improved following @dillaman suggestions